determined-ai/determined
# fix: replace div with table in Trial Comparison View

## Description
[WEB-1405] [WEB-1401]

### Before

<img width="1728" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/15af15b4-bb03-417a-bb14-54cd988eab1f">



### After

<img width="1728" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/d104479b-d1a0-4189-896c-78ca17d03766">





## Test Plan
- Check if Trial Comparison View uses table element instead of div element
- Check if the table is scrollable horizontally and vertically if there are multiple experiments are selected
  - The first row and column should be sticky
- Check if `Metrics` and `Hyperparameters` filter work as before
- Check if there's no regressions




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1405]: https://hpe-aiatscale.atlassian.net/browse/WEB-1405?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WEB-1401]: https://hpe-aiatscale.atlassian.net/browse/WEB-1401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
